adapters/risc0/src/host.rs

```diff
@@ -16,8 +20,18 @@ pub struct Risc0Host<'a> {
 
 impl<'a> Risc0Host<'a> {
     pub fn new(elf: &'a [u8]) -> Self {
+        #[cfg(not(feature = "bench"))]
```

Comment:
We should have another method `new_for_bench` and ` #[cfg(not(feature = "bench"))]` the whole method